tests/end2end-tests/data_gen/process_movielens.py

```diff
@@ -90,6 +90,11 @@ def write_data_parquet(data, data_file):
 edge_data = {'src_id': edges[0], 'dst_id': edges[1], 'rate': edges[2]}
 write_data_parquet(edge_data, '/data/ml-100k/edges.parquet')
 
+# generate data for homogeneous optimization test
+edges = pandas.read_csv('/data/ml-100k/u.data', delimiter='\t', header=None)
+edge_data = {'src_id': edges[1], 'dst_id': edges[1], 'rate': edges[2]}
+write_data_parquet(edge_data, '/data/ml-100k/edges_homo.parquet')
```

Comment:
We can also generate the json config in process_movielens.py

Reply:
For our other gconstruct test cases, we directly use static json file stored in ```end2end-tests/data_gen```. We can keep the same way as what we did before.